omero/developers/Matlab.txt

```diff
@@ -9,12 +9,11 @@ See |DevelopingOmeroClients| and |OmeroModel|, for an introduction to
 Installing the OMERO.matlab toolbox
 -----------------------------------
 
--  Download the latest released version from the
-   :downloads:`download page <>`.
+-  Download the latest released version from the :downloads:`download page <>`.
```

Comment:
feel free to ignore, as not this PR's fault and it's a rebase, but "downloads page" plural probably better fits the page title and functionality